### PR TITLE
Load worker as a normal synchronous module

### PR DESCRIPTION
**Pull request by phamann**
**⚠️ Note this is a PR into the v.2.0.0 branch and not master**

### TL;DR
Due to the cross-origin loading restrictions of `WebWorkers` this refactors the scout script to load the worker library as a normal synchronous module inline, instead of instantiating a new `WebWorker` with the library. This is hopefully a temporary measure until we can iron out the related issues. 

### Why?
I was unaware of the tighter same-origin restrictions of WebWorkers, regardless of CORS headers or the normal cross-origin rules within browsers. Therefore, when v2.0.0 of the library was loaded as a 3rd party we were observing errors stating the worker was prevented from loading due to cross-origin restrictions.

It is possible to load cross origin web workers via an inline `Blob` URL, such as  `new Worker(URL.createObjectURL(new Blob(script))`. However, this requires the `blob:` scheme to be allowed via the host documents `Content-Security-Policy`, which isn't currently the case for some of our production users.

I hope for this to be a temporary stop-gap until we decide a path forward with workers. I am currently considering making the feature opt-in and available via a query parameter, which if present will load the library as a cross-origin worker via `Blob` and dynamically load as main-thread module if not.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -388,6 +388,15 @@
         "@babel/helper-plugin-utils": "^7.0.0"
       }
     },
+    "@babel/plugin-syntax-dynamic-import": {
```
Comment (clight-fastly) [inline package-lock.json]: Is this currently used?

Comment (phamann) [reply package-lock.json]: Hmmm. I must have manually removed from the package.json file and thus the lock wasn't cleaned up. I will fix and rebase.

Comment (phamann) [reply package-lock.json]: Reolved
